.ci/docker/common/install_onnx.sh

```diff
@@ -20,6 +20,7 @@ pip_install \
 
 pip_install coloredlogs packaging
 pip_install onnxruntime==1.23.1
+pip_install onnx-ir==0.1.12
 pip_install onnxscript==0.5.4
```

Comment:
Why do we need those in two places? If it's in requirements-ci it could safely be removed from here...